Source/Editor/Windows/Assets/SpriteAtlasWindow.cs

```diff
@@ -291,6 +295,9 @@ public override void Save()
 
             ClearEditedFlag();
             _item.RefreshThumbnail();
+
+            _properties.UpdateSprites();
+            _propertiesEditor.BuildLayout();
```

Comment:
This is actually added for a separate issue the editor has: When saving the list of sprites, names were not being changed and new ordering was not being applied. This fixes the inconsistency.